doc/cygnus-ngsi/flume_extensions_catalogue/last_data_function.md

```diff
@@ -111,9 +137,9 @@ A query like this one is executed
 ### POSTGRESQL
 
 `INSERT INTO pruebapostmanx.subpruebapostman_5dde9_wastecontainer_last_data 
-(recvTime,fiwareServicePath,entityId,entityType,fillingLevel,fillingLevel_md) VALUES (?, ?, ?, ?, ?, ?) 
+(entityId,entityType,fiwareServicePath,recvTime,fillingLevel,fillingLevel_md) VALUES (?, ?, ?, ?, ?, ?) 
 ON CONFLICT (entityId) DO UPDATE SET 
-recvTime=EXCLUDED.recvTime, fiwareServicePath=EXCLUDED.fiwareServicePath, entityType=EXCLUDED.entityType, 
+entityType=EXCLUDED.entityType, fiwareServicePath=EXCLUDED.fiwareServicePath, recvTime=EXCLUDED.recvTime, 
 fillingLevel=EXCLUDED.fillingLevel, fillingLevel_md=EXCLUDED.fillingLevel_md 
 WHERE pruebapostmanx.subpruebapostman_5dde9_wastecontainer_last_data.entityId=EXCLUDED.entityId 
 AND to_timestamp(pruebapostmanx.subpruebapostman_5dde9_wastecontainer_last_data.recvTime, 'YYYY-MM-DD HH24:MI:SS.MS') 
```

Comment:
This documentation has been checked using this conf:

```
cygnus-ngsi.sinks.postgis-type-database-sink-lastdata.last_data_unique_key = entityId
```

And getting this from traces:

```
INSERT INTO urbodeployer.palenciacicloagua_waterpump_lastdata
(entityId,entityType,fiwareServicePath,recvTime,probingFlowMeter,batteryMinThreshold,wellFlowMeter,entityid,battery,TimeInstant,pumpStatus,batteryMaxThreshold) VALUES (....)
ON CONFLICT (entityId) DO UPDATE SET
entityType=EXCLUDED.entityType,fiwareServicePath=EXCLUDED.fiwareServicePath,recvTime=EXCLUDED.recvTime,
probingFlowMeter=EXCLUDED.probingFlowMeter,batteryMinThreshold=EXCLUDED.batteryMinThreshold,wellFlowMeter=EXCLUDED.wellFlowMeter,entityid=EXCLUDED.entityid,battery=EXCLUDED.battery,TimeInstant=EXCLUDED.TimeInstant,pumpStatus=EXCLUDED.pumpStatus,batteryMaxThreshold=EXCLUDED.batteryMaxThreshold
WHERE urbodeployer.palenciacicloagua_waterpump_lastdata.entityId=EXCLUDED.entityId
AND to_timestamp(urbodeployer.palenciacicloagua_waterpump_lastdata.recvtime::text, 'YYYY-MM-DD HH24:MI:SS.MS') < to_timestamp(EXCLUDED.recvtime::text, 'YYYY-MM-DD HH24:MI:SS.MS')
```

Reply:
A slight different in the last line has been fixed in this commit cf485d8804688851eb70cee4f881cbae60043e74